bundles/org.openhab.binding.lifx/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -45,4 +48,17 @@
 		</parameter>
 	</config-description>
 
+	<config-description uri="channel-type:lifx:effect">
+		<parameter name="effectMorphSpeed" type="decimal" min="0" max="25" step="0.1" required="false" unit="s">
```

Comment:
In the [protocol](https://lan.developer.lifx.com/docs/firmware-effects#section-settileeffect-719) the `speed` is an unsigned 32-bit integer representing milliseconds. Wouldn't it make sense to increase the max value for users who want a really slow effect?

Reply:
I took the max value from the LIFX app on IOS. There is no limitation, I will increase it to 120.